docs/api/commands/session.mdx

```diff
@@ -322,7 +322,7 @@ cy.session('user', () => {
 ### Caching session data across specs
 
 If you want to use the same session across multiple specs in the same Cypress
-run on the same machine, add `cacheAcrossSpecs=true` to the session options to
+run on the same machine, add `cacheAcrossSpecs:true` to the session options to
```

Comment:
@Lucas-Athayde I am not entirely sure this is a typo, rather trying to display that the value should be set to true. Do we know if this is consistent with Cypress' current documentation style?

Reply:
Hey there @AtofStryker, thank you for your comment and reviewing my PR.
You've raised a good point, after checking out other parts of the documentation I could see that you're indeed correct!
It's actually displaying the value that is has to be set to.
Sorry for any inconvenience.